src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -82,6 +82,7 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
             const listOfDatabases: DatabaseListResult = await client.databases.listByServer(nonNullProp(this, 'resourceGroup'), nonNullProp(this, 'azureName'));
             dbNames = listOfDatabases.map(db => db?.name);
         } else if (this.connectionString.databaseName) {
+            await getClientConfig(this, this.connectionString.databaseName);
```

Comment:
Without this getClientConfig check, the attached database-specific account will not throw "Must enter credentials..." but would direct to a prompt to enter credentials. The enter credentials prompt won't work with the current setup.

Reply:
I'm confused by this comment - is it still accurate? We don't want "Must enter credentials..." to show up because they're optional for attached scenarios

Reply:
As connectionString is a ServerTreeItem property, databaseName would only be present for the attached scenario. But this modification is unnecessary change for this particular PR, I will remove it here and include it in the other one.